src/main/webapp/app/service/firebase/firebase-gene-service.ts

```diff
@@ -392,8 +392,14 @@ export class FirebaseGeneService {
     initialUpdate?: boolean,
   ) => {
     const { hugoSymbol } = parseFirebaseGenePath(rctPath);
+
+    if (!uuid) {
+      uuid = generateUuid();
+    }
+
     if (initialUpdate) {
       return this.firebaseRepository.create(rctPath, newRelevantCancerTypes).then(() => {
+        this.firebaseRepository.create(`${rctPath}_uuid`, uuid);
```

Comment:
Can probably just bundle this in with the update

Reply:
I should really also bundle the `_review` with the update too. I need to parse `rctPath` to get the Implication path in order to update. The `rctPath` given by the RCT Modal is something like `Genes/BRAF/mutations/0/tumors/0/TIs/0/treatments/0/excludedRCTs`, but updating would require the path to the treatment.

Reply:
I already made this change in my multi-location update PR, so I will leave the code for now if that's okay.

Reply:
Yeah for sure, makes sense to me